test/unit/mongodb/mongodb-connectionoptions-test.js

```diff
@@ -75,8 +75,8 @@ describe('dbConn.configureDb', function () {
                 expected: {
                     url: 'mongodb://example.com:27017/' + dbConn.DEFAULT_DB_NAME,
                     options: {
-                        useNewUrlParser: true,
-                        useUnifiedTopology: true
+                        // useNewUrlParser: true,
+                        // useUnifiedTopology: true
```

Comment:
I understand this kind of leftovers (several ones in the PR) should be removed

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1676/commits/144d5686e70e190e5bbb176ba0f1555573ce07ad